packages/cli/src/util/emoji.ts

```diff
@@ -21,3 +21,10 @@ export function prependEmoji(message: string, emoji?: string): string {
 
   return message;
 }
+
+export function removeEmoji(message: string): string {
+  const regExp = new RegExp(Object.values(emojiLabels).join('|'), 'gi');
```

Comment:
Since `emojiLabels` is const, the `regExp` can be initialized and cached outside `removeEmoji()` so that the regex doesn't need to be reconstructed and compiled every time `log()` is called. Should probably also give it a better name like `stripEmojiRegex` or something.